plugins/woocommerce-admin/client/marketing/components/CollapsibleCard/CollapsibleCard.tsx

```diff
@@ -20,18 +20,20 @@ import './CollapsibleCard.scss';
 
 type CollapsibleCardProps = {
 	header: string;
-	className: string;
 	children: React.ReactNode;
+	className?: string;
 	footer?: React.ReactNode;
+	initialCollapsed?: boolean;
```

Comment:
💅 Super minor. Since initialCollapsed is false by default. I guess it could be safe to use `initialCollapsed: boolean` without optional flag.

Reply:
Without the optional flag `?`, it would mean that `initialCollapsed` is a required field, and TypeScript will tell us error in places where we don't specify `initialCollapsed`, such as in `InstalledExtensions` component:

https://github.com/woocommerce/woocommerce/blob/9a5ebc450a6aa5aab6dd070afa5687aa6aa3721c/plugins/woocommerce-admin/client/marketing/overview-multichannel/InstalledExtensions.tsx#L86-L89

Reply:
Right!